kinesis-video-pic/src/client/src/StreamEvent.c

```diff
@@ -686,9 +686,23 @@ STATUS streamTerminatedEvent(PKinesisVideoStream pKinesisVideoStream, UPLOAD_HAN
                             (pUploadHandleInfoItem->state & UPLOAD_HANDLE_STATE_ACTIVE) != UPLOAD_HANDLE_STATE_NONE) {
                             otherActiveSessionPresent = TRUE;
                             pKinesisVideoStream->connectionState = UPLOAD_CONNECTION_STATE_NOT_IN_USE;
+                            // dont spawn new session since there is already an active one.
                             spawnNewUploadSession = FALSE;
-                        }
 
+                            DLOGW("Upload handle %" PRIu64 " terminated with result %u while other upload handles are active."
+                                "Data sent through upload handle %" PRIu64 " may not be fully persisted.",
```

Comment:
Are we referring "data sent through" as one fragment size bytes or can it be multiple fragments?

Reply:
in most of the cases, it should be one fragment.

Reply:
Thanks for adding this. Suggest rewording to stating that the "last fragment for upload handle blah might not be fully persisted". Also, output the fragment timestamp like when calling the error callback